Content.Server/MachineLinking/Models/SignalPort.cs

```diff
@@ -1,81 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Diagnostics.CodeAnalysis;
-using Content.Server.MachineLinking.Events;
-using Content.Server.MachineLinking.Exceptions;
-using Robust.Shared.Serialization.Manager.Attributes;
-
-namespace Content.Server.MachineLinking.Models
-{
-    [DataDefinition]
-    public sealed class SignalPort
-    {
-        [DataField("name", required: true)] public string Name { get; } = default!;
-        [DataField("type")] public Type? Type { get; }
-        /// <summary>
-        /// Maximum connections of the port. 0 means infinite.
-        /// </summary>
-        [DataField("maxConnections")] public int MaxConnections { get; } = 0;
-
```

Comment:
I guess this is technically a loss in functionality, but I really don't mind. If someone wants to wire up the whole station to a single light switch I'm fine with that